server/src/main/scala/slickdb/UserTable.scala

```diff
@@ -49,13 +54,16 @@ case class UserTable(tables: Tables) extends UserTableLike {
       }
   }
 
-  def insertOrUpdateUser(user: LoggedInUser, inactive_email_sent: Option[java.sql.Timestamp], revoked_access: Option[java.sql.Timestamp])(implicit ec: ExecutionContext): Future[Int] = {
-    tables.run(userTableQuery.insertOrUpdate(UserRow(user.id, user.userName, user.email, new Timestamp(new Date().getTime), inactive_email_sent, revoked_access)))
-      .recover {
-        case throwable =>
-          log.error(s"insertOrUpdate failed", throwable)
-          0
-      }
+  def insertOrUpdateUser(user: LoggedInUser,
+    inactive_email_sent: Option[java.sql.Timestamp],
```

Comment:
Can we keep indentation consistent so the lines below start in line with the first parameter?